src/specialOffer/SpecialOffer.story.mdx

```diff
@@ -0,0 +1,198 @@
+import { action } from '@storybook/addon-actions'
+import { ArgsTable, Canvas, Meta, Story } from '@storybook/addon-docs/blocks'
+
+import { ContentDivider } from '../divider/contentDivider'
+import { ItemsList } from '../itemsList'
+import { BaseSection } from '../layout/section/baseSection'
+import { SpecialOffer } from './index'
+
+<Meta title="Widgets/SpecialOffer" />
+
+# **SpecialOffer**
```

Comment:
Hey, I was thinking that this component looks a lot like a Thumbnails/ Card pattern component. Do you think we could rename it ?!

https://getbootstrap.com/docs/5.0/components/card/
https://getbootstrap.com/2.2.0/components.html#thumbnails

Reply:
The internals look very similar but I'm not confortable renaming it.

In my mind a good Thumbnails/Card component is generic enough to be used in all situations with slots/placeholder that are agnostics of their real content.

This is not the situation here. You can see with the typings of the component that it is very locked to the need of the Good Deals page (0 or 1 informationLink), picture cannot be a `JSX.Element`, 1 or 2 links or 1 or 2 buttons, special size for the image. Those are specificities that we don't want for a generic Thumbnails/Card component I think.

I'm not saying a Thumbnails/Card component is a bad idea but that maybe there is two levels?
- a generic Thumbnails/Card component that we can use here and elsewhere. This one would be internal to kirk.
- more high level exposed components that embed business rules but used the generic component internally

What do you think? I was talking with Michael about this this morning and my feeling is that it's not a straightforward idea for the SPA team. Maybe kirk is not thought this way at all and that I'm not aligned with what it should expose.

Reply:
may we take that to chapter / kick discution since we have a lot of people involved?

Reply:
Yup I think it's an important discussion but one that couldn't be just be done in a meeting. The best way to do in mind mind would be to open an ADR document for review.